src/tests/Equinor.ProCoSys.IPO.Query.Tests/GetInvitationsQueries/GetInvitations/GetInvitationsQueryHandlerTests.cs

```diff
@@ -829,13 +887,31 @@ public async Task Handler_ShouldSeparateOutAdditionalConstructionCompanyParticip
             }
         }
 
+        [TestMethod]
+        public async Task Handler_ShouldNotShowParticipants_WhenUsingPersonalEmail()
```

Comment:
I'm having a bit of trouble verifying that this test is testing what the method name says.

"Should not show participants" -> it seems you are testing AdditionalConstructionCompanyReps for being 0, what is the relationship between participants and AdditionalConstructionCompanyReps?

"WhenUsingPersonalEmail" -> where do you set up that personal email is used in the test?

Reply:
@jsoi-eq 
See [](https://statoildeveloper.visualstudio.com/Statoil.NextGenProCoSys/_workitems/edit/110737) for details about where/ how the use personal email flag is set. In this test, the flag is set in the method SetupNewDatabase.
"no names should be presented in search results for roles where user personal email is set to true".
 In this test AdditionalConstructionCompanyReps is used as an example to prove the logic. Relationship between participants and AdditionalConstructionCompanyReps is one to one. It is a filtered representation of participants. As defined by GetAdditionalConstructionCompanyReps.